packages/compiler/entrypoints/server.js

```diff
@@ -1,5 +1,5 @@
 /**
- * File serving as an entrypoint to resolve a local cadl install from a global install.
+ * File serving as an entrypoint to resolve a local typespec install from a global install.
```

Comment:
I think we're going to break this anyway now and maybe we can delete this now.

Reply:
Not sure how the change should be here. Let's chat

Reply:
I was wrong here, don't change anything in entrypoints/. The only file that is not useful anymore is compiler/core/cli.ts that is marked "for backwards compatibility" in comment because we can't be backwards compatible to global `cadl` finding local `tsp`.